### PR TITLE
doc updates

### PR DESCRIPTION
**Pull request by chook**

### CODE REVIEW

File: services.yaml [hunk 1]
```diff
@@ -1573,17 +1579,17 @@ paths:
...
           $ref: "#/responses/ForbiddenError"
-  /services/{service_id}/views/{view_id}/events:
+  /services/{env_id}/views/{view_id}/events:
     get:
       tags:
         - Views
         - Events
       summary: Fetch events details
-      description: Fetch events data in a selected timeframe
+      description: Fetch events data for a target timeframe, application / server / deployment set.
```
Comment (jenjenp) [inline services.yaml:1632]: not sure i got the addition of application / server / deployment. do you mean the ability to filter (which as i see not exists)?

File: services.yaml [hunk 2]
```diff
@@ -2045,12 +2051,12 @@ paths:
     post:
       tags:
         - Labels
-      summary: "Create new label"
-      description: "Create new label"
+      summary: "Define new label"
```
Comment (jenjenp) [inline services.yaml]: I think that 'create' is better than 'define'. 'Define' doesn't feel like creating something new and sounds more as 'customize existing'

**Review by jenjenp — DISMISSED** [review]

File: services.yaml (hunk 1 above)
Comment (chook) [reply services.yaml:1632]: it does exist:
```
  - in: query
           name: server	           name: server
           type: string	           type: string
           description: Server(s) added to the filter	           description: Server(s) added to the filter
         - in: query	         - in: query
           name: app	           name: app
           type: string	           type: string
           description: Application(s) added to the filter	           description: Application(s) added to the filter
         - in: query	         - in: query
           name: deployment	           name: deployment
           type: string	           type: string
           description: Deployment(s) added to the filter	           description: Deployment(s) added to the filter
         - in: query	         - in: query
```

File: services.yaml (hunk 2 above)
Comment (chook) [reply services.yaml]: i agree

**Pull request merged**